### PR TITLE
eroshare.com host file added

### PR DESCRIPTION
**Pull request by Cervenka**
As requested here: https://www.reddit.com/r/Enhancement/comments/3oyyax/feature_request_inline_view_for_erosharecom/

### CODE REVIEW

File: lib/modules/hosts/eroshare.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+		} else {
+			def.reject();
+		}
+
+		return def.promise();
+	},
+	handleInfo: function(elem, info) {
+		elem.type = 'IFRAME';
+		elem.setAttribute('data-embed', info);
+    elem.setAttribute('data-width','550');
```
Comment (andytuba) [inline lib/modules/hosts/eroshare.js]: whitespace -- tabs vs spaces

**Comment by andytuba** [comment]
neato, thanks for stepping up to contribute this!

**Comment by Cervenka** [comment]
Hi @andytuba, I included an additional host-file for uploadly.com. I hope I got the tabs right this time and that this is in time for the next release. Thank you!

**Comment by andytuba** [comment]
Merged to release/4.6.1

I'll comment again on this PR when it's released.

**Comment by andytuba** [comment]
v4.6.1 rolling out now!

**Pull request closed without merging**